website/docs/r/linux_virtual_machine_scale_set.html.markdown

```diff
@@ -268,25 +268,33 @@ An `automatic_instance_repair` block supports the following:
 
 * `enabled` - (Required) Should the automatic instance repair be enabled on this Virtual Machine Scale Set?
 
-* `grace_period` - (Optional) Amount of time (in minutes, between 30 and 90) for which automatic repairs will be delayed. The grace period starts right after the VM is found unhealthy. The time duration should be specified in ISO 8601 format. Defaults to `PT30M`.
+* `grace_period` - (Optional) Amount of time for which automatic repairs will be delayed. The grace period starts right after the VM is found unhealthy. Possible values are between `10` and `90` minutes. The time duration should be specified in `ISO 8601` format (e.g. `PT10M` to `PT90M`).
+
+-> **Note:**  Once the `grace_period` field has been set it cannot be removed.
```

Comment:
Nit-picking here but strictly speaking this isn't correct. Since the property has `Computed` set users can remove this from their config without hitting any API limitations or experiencing a diff so I think this should be removed, or perhaps rephrased?

Reply:
Fixed.

Reply:
Not sure if a commit is missing here but these have not been changed?

Reply:
I believe I misunderstood what you were saying, now that I have read it a few more times, what you want is for the note to be removed since if you remove it from the config is will just pull it from state?